spec/std/log/metadata_spec.cr

```diff
@@ -28,6 +28,14 @@ describe Log::Metadata do
     m({a: 1}).extend({} of Symbol => String).should_not be_empty
   end
 
+  describe "#dup" do
+    it "creates a shallow copy" do
+      Log::Metadata.empty.dup.should eq(Log::Metadata.empty)
```

Comment:
Shouldn't this use:

```suggestion
      Log::Metadata.empty.dup.should be(Log::Metadata.empty)
```

Reply:
I think this would be testing `.empty` more than `#dup`... But yeah technically, all expectations could use `be`. But I think that's actually an implementation detail. It's not strictly necessary for `#dup` to return the same instance, but it does that for performance reasons.

Reply:
It isn't part of `#dup`'s contract to preserve object identity of immutable reference types, one day `#dup` might create a shallow copy for other reasons. So `eq` suffices for unit tests of `#dup`